BuildingBlocks/src/SpecFlowCucumberResultsExporter/Extensions/Exporter.cs

```diff
@@ -6,18 +6,17 @@ public class Exporter : ReportingStepDefinitions
     public static void ExportToCucumber(string path = default, string fileName = default)
     {
         path ??= "../../../TestResults/";
-        fileName ??= $"specflow_cucumber_{DateTimeOffset.UtcNow:yyyyMMdd}";
-        fileName += ".json";
+        fileName ??= $"specflow_cucumber_{DateTimeOffset.UtcNow:yyyyMMdd}.json";
 
         Reporters.Add(new JsonReporter());
 
         Reporters.FinishedReport += (sender, args) =>
         {
-            var file = new System.IO.FileInfo(path);
+            var file = new FileInfo(path + fileName);
```

Comment:
You should probably use Path.Combine instead of string concatination. Because if you forget to pass a trailing slash in the `path` parameter, you end up with somethink like ",../../../myfoldermyfilename.json"